packages/hosted-form-v2/src/payment/manual-order-payment-request-sender.ts

```diff
@@ -42,6 +42,6 @@ export class ManualOrderPaymentRequestSender {
             },
         };
 
-        await this._requestSender.post<void>(`${this._paymentOrigin}/payments`, options);
+        return this._requestSender.post<unknown>(`${this._paymentOrigin}/payments`, options);
```

Comment:
Interesting fix, are we going to be validating the response in client implementations?

Reply:
No, we validate in payment handler, and post events only when we receive specific type https://github.com/bigcommerce/checkout-sdk-js/pull/2631/files#diff-2d86a89846731279b0343d2dde3dad8a5de52e16c8a547dc40b40408bd15ec5f

Reply:
Similar to the payments api implementation [here](https://github.com/bigcommerce/checkout-sdk-js/blob/b445659272efda3f381f1ed7332d8adef5da566d/packages/core/src/payment/strategies/ppsdk/step-handler/step-handler.ts#L16)